src/main/scala-2.12/com/typesafe/sbt/PluginCompat.scala

```diff
@@ -0,0 +1,22 @@
+package com.typesafe.sbt
```

Comment:
```suggestion
package com.typesafe.sbt.jse
```

This should be in a unique package to avoid collision.